app/components/CaseStudyGallery.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
```

Comment:
The React import isn't necessary here. Please be sure to remove any unused imports. Thanks!

Reply:
Removed. Sorry I keep missing these